.classpath

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I prefer to leave the Eclipse files.

Reply:
ok

Reply:
Ok I rolled back, but since I use m2e integration, my .classpath and .project are a bit different from yours...
Maven used to be a bit of a pain, but m2e works beautifully for simple builds like this one, you hardly notice the difference in eclipse.

Reply:
So it does hurt, in a way ;-) depending on whether or not you use maven integration in eclipse.
Let me know if this m2e enabled .project and .classpath work for you, or if you really need to stick to the eclipse build model.